Middleware/index.js

```diff
@@ -7,7 +7,7 @@ import express from 'express';
 const Middleware = (app) => {
 	app.use(bodyParser.json());
 	//public the view
-	app.use(express.static('View'));
+	//app.use(express.static('View'));
```

Comment:
please delete commented code if you no longer need it, don't push commented code online